BitwardenShared/UI/Platform/Settings/Settings/AutoFill/AutoFillView.swift

```diff
@@ -14,19 +14,42 @@ struct AutoFillView: View {
 
     var body: some View {
         VStack(spacing: 20) {
+            autofillActionCard
+
             autoFillSection
 
             additionalOptionsSection
         }
+        .animation(.easeInOut, value: store.state.badgeState?.autofillSetupProgress == .complete)
         .scrollView()
         .navigationBar(title: Localizations.autofill, titleDisplayMode: .inline)
         .task {
             await store.perform(.fetchSettingValues)
         }
+        .task {
+            await store.perform(.streamSettingsBadge)
+        }
     }
 
     // MARK: Private views
 
+    /// The action card for setting up autofill.
+    @ViewBuilder private var autofillActionCard: some View {
+        if let progress = store.state.badgeState?.autofillSetupProgress, progress != .complete {
```

Comment:
same comment here, just to move logic out of the view.  feel free to ignore tho.